CONTRIBUTING.md

```diff
@@ -59,18 +59,25 @@ The `metadata.json` file contains metadata of the module including `name`, `desc
   // The name of the module (10 - 60 characters).
   "name": "Sample module",
   // The description of the module (10 - 1000 characters).
-  "description": "Sample module description",
+  "summary": "Sample module description. This should be a short description of the functionality of the module. A more detailed description can be included in the README.md file.",
```

Comment:
Could you also help update $schema to v2?

Reply:
done